apis/resources/logging/v1beta1/logmetric_types.go

```diff
@@ -217,7 +217,7 @@ type LoggingLogMetricStatus struct {
 
 	/* Output only. The last update timestamp of the metric. This field may not be present for older metrics. */
 	// +optional
-	UpdateTime *metav1.MicroTime `json:"updateTime,omitempty"`
+	UpdateTime *string `json:"updateTime,omitempty"`
```

Comment:
Yeah, we need to make this a convention.

Reply:
Right, we should probably have a test - that would be much more reliable than a doc.  However, I don't know what the rule would be... if the CRD previously existed, we should not have a metav1.Time or metav1.MicroTime, I guess?